CONTRIBUTORS.md

```diff
@@ -10,3 +10,6 @@
 - Simon Barendse [@simonbarendse](https://github.com/simonbarendse)
 - Valerio Barrila [@ninjatux](https://github.com/ninjatux)
 - Floris van der Grinten [@florisvdg](https://github.com/florisvdg)
+- Márton Soos [@Marton6](https://github.com/marton6)
+- Teddy Filip [@edif2008](https://github.com/edif2008)
```

Comment:
Hi @edif2008, you have made it to the contributor list! :smile:

![](https://media2.giphy.com/media/W8krmZSDxPIfm/giphy.gif)

Do you want this to say Teddy or Eduard?

Reply:
Feel free to add a commit to this branch if you want it changed

Reply:
Changed to Eduard to be professional. 😄

Reply:
Then I'd suggest changing your profile to Eduard as well 😉

Reply:
Done.